File: 240328/007/007.java
import java.util.Scanner;
public class Main {
    public static class cr {
        String secret_code;
        char alpha;
        int time;
        public cr(String secret_code, char alpha, int time) {
            this.secret_code = secret_code;
            this.alpha = alpha;
            this.time = time;
        }
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        cr cr1 = new cr("ab",'a',0);
        String secret_code = sc.next();
        char alpha = sc.next().charAt(0);
        int time = sc.nextInt();
        cr1.secret_code = secret_code;
        cr1.alpha = alpha;
        cr1.time = time;
        System.out.printf("secret code : %s\n", secret_code);
        System.out.printf("meeting point : %c\n", alpha);
        System.out.printf("time : %d", time);
    }
}

File: 240417/007/007.kt
import java.util.Scanner

data class Letter(
    val secretCode: String,
    val meetingPoint: String,
    val time: Int
)

fun main() {
    val sc = Scanner(System.`in`)

    val secretCode = sc.next()
    val meetingPoint = sc.next()
    val time = sc.nextInt()

    Letter(secretCode, meetingPoint, time).let {
        println("secret code : $secretCode")
        println("meeting point : $meetingPoint")
        print("time : $time")
    }
}